Modules/@babylonjs/react-native/windows/scripts/Utils.psm1

```diff
@@ -63,7 +69,7 @@ function Compile-Solution {
     )
 
     $MSBuild = Get-MSBuildPath
-    & "$MSBuild" /p:Configuration="$Configuration" /p:Platform="$Platform" $Solution
+    & "$MSBuild" /p:Configuration="$Configuration" /p:Platform="$Platform" /m:32 $Solution

```

Comment:
Should this just be `/m`? My understanding is that that tells MSBuild to build in parallel, but lets it decide how much parallelism based on the number of cpu cores in the machine where it is actually building.

Reply:
Also, within a single project, the vc++ compiler can compile c++ files in parallel with the `/MP` compiler switch. Can/do/should we use that as well?

Reply:
specifying processors was more confusion on my end. If msbuild can auto determine the correct number of processors, I would prefer that as well.

/MP is already set through cmake for majority if not all BabylonNative static libs. It is also set for BabylonReactNative.dll

Reply:
removed the number of processors from m

Reply:
FYI, you might want to consider enabling parallelism on the project level too. Looks like there's been some improvements added in Visual Studio 2019 16.3 - https://devblogs.microsoft.com/cppblog/improved-parallelism-in-msbuild/

Reply:
Parallelism feels beyond the scope of this review. I've filed an issue here to track: https://github.com/BabylonJS/BabylonNative/issues/584